generated/backend/EmailAddress.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Do not edit this file it is auto-generated by italia-utils / gen-api-models.
+ * See https://github.com/teamdigitale/italia-utils
+ */
+/* tslint:disable */
+
+import { EmailString } from "italia-ts-commons/lib/strings";
```

Comment:
you don't need this generated type, you can use the `EmailString` type instead of `EmailAddress`, they are the same type

Reply:
fixed in c5dd8fc